app/templates/components/quiz-module.hbs

```diff
@@ -1,7 +1,5 @@
-<div class="row" style="margin-left:0px; margin-right:10px">
-    <div style="margin-bottom: 20px">
-        {{{module.description}}}
-    </div>
+<div style="margin-bottom: 20px">
+    {{{module.description}}}

```

Comment:
Nedaří se mi zjistit, jakým způsobem tato změna rozbila zobrazení radio boxu a labelu na jednom řádku.
https://kyzikos.fi.muni.cz/ulohy/130
Nedělal jsi nějakou změnu na backendu?

Reply:
Tohle není problém ani backend, ani frontendu, je to problém textu modulu. Už je to opraveno, web totiž číslo intepretoval jako <ol>.